src/docs/integrations/jira/index.mdx

```diff
@@ -1,18 +1,27 @@
 ---
-title: "Jira Integration"
+title: 'Jira Integration'
 ---
 
-Create an [Atlassian Cloud Instance](http://go.atlassian.com/cloud-dev)
+Create an [Atlassian Cloud Developer Instance](http://go.atlassian.com/cloud-dev)
 
-Configure Jira settings.
-  1. Navigate to Jira settings (cog icon) > Apps > Manage apps.
-  2. Scroll to the bottom of the Manage apps page, and click Settings (1).
-  3. Select Enable development mode and Enable private listings (2), and click Apply.
+Atlassian offers a development-specific org option, different from simply having a personal Jira organization. Even if you already have a personal Jira org, it's worth creating a development org, because it will correctly set a number of necessary options for you automatically.
```

Comment:
What are the options? I use a regular personal Jira organization.

Reply:
A Jira org existing, users being in projects, and users being "assignable" (which also makes them eligible to be the reporter, which is a required field in our Create Issue modal). 

It's partially my fault, because - thinking I already had a personal Jira - I didn't click the Atlassian Cloud link. But trying to log into Jira with my personal email took me instead to BitBucket (because it turns out I didn't have a personal Jira account), and so first I wandered around the Atlassian admin figuring out how to add Jira. Then I spent more time than I care to admit Friday afternoon tearing my hair out, trying to figure out why on earth I wasn't showing up in the options for the Reporter field even though it was my personal org, I'm the one who created the linked project, and the basic member search API would return my record just fine. Turns out I had to both add myself to the project and go into project settings and change the permissions for "assignable users," but it took a lottttt of trial and error and print debugging and hitting various API endpoints before I finally figured that out.

So then Monday morning, I popped open this repo to add a note about the necessary steps, and noticed that the link I'd ignored on Friday said "cloud-dev" at the end, which clued me into the fact that this was probably different than just a personal Jira. So I started again, this time making a dev org (totally separate from my personal one), and a) Jira was already there and didn't need to be turned on, b) when I created a project I was automatically added to it, and c) the permissions by default were set correctly such that I was assignable and therefore showed up in the Reporter field in our modal. So instead of the literal hours it took me on Friday, I was instead able to set up the integration in under five minutes - everything "just worked" - and so I figured that's why we chose that option instead of just telling people "go sign up for Jira."

Given our conversation on Slack, though, I'm taking it you didn't need to go through all of that rigamarole when you used your personal Jira?

Reply:
😅 I don't remember a similar rigamarole, though I did set it up quite some time ago. I hadn't actually heard of the dev specific org option which is why I was curious. It may have not existed when we (I? did I write these?) wrote these docs.